### ISSUE TITLE
option to disable all auto-capitalization

### ISSUE THREAD

**Issue by EmeraldSnorlax** [dslul/openboard#837]
**Is your feature request related to a problem? Please describe.**
i almost never want anything i type that isn't formal to be "properly" capitalised. this also seems to be a common sentiment with younger people in general in the circles im in -- having "correct" caps comes off as "trying too hard".

choosing what case you use has a significant impact on the tone and personality of your message, and is becoming an intentional choice more than ever before, so it would be nice if openboard trusted your intent with it.

**Describe the solution you'd like**
an option to disable all suggestions and replacements that adjust case -- including proper nouns, abbreviations, and "i" (first person pronoun). if correction is needed, it should correct spelling but preserve the case the word was written in -- whether that's all caps, all lower, or "correct" sentence-casing (stretch: "mOcKiNg CaSe" too?)

the current options only allow disabling uppercase on the first character after a full stop.

**Describe alternatives you've considered**

hitting backspace on every suggestion that auto capitalises, manually changing the text

**Additional context**

maybe related to #681?

https://www.youtube.com/watch?v=fS4X1JfX6_Q

tl;dr: case has little meaning other than what we agree it does, and for many people, playing with casing is an intentional choice.
